### PR TITLE
pkg/prometheus: add more logs to syncNodeEndpoints()

### PR DESCRIPTION
**Pull request by simonpasquier**
Related to #2733

### CODE REVIEW

File: pkg/prometheus/operator.go [hunk 1]
```diff
@@ -579,6 +579,8 @@ func getNodeAddresses(nodes *v1.NodeList) ([]v1.EndpointAddress, []error) {
 }
 
 func (c *Operator) syncNodeEndpointsWithLogError(ctx context.Context) {
+	level.Debug(c.logger).Log("msg", "syncing nodes into Endpoints object")
```
Comment (lilic) [inline pkg/prometheus/operator.go:582]: Nit: 
```suggestion
	level.Debug(c.logger).Log("msg", "Syncing nodes into Endpoints object")
```

**Review by lilic — APPROVED** [review]
lgtm one nit

File: pkg/prometheus/operator.go (hunk 1 above)
Comment (simonpasquier) [reply pkg/prometheus/operator.go:582]: done

**Review by lilic — APPROVED** [review]
Still lgtm 👍

**Pull request merged**